userpanel/modules/info/style/bclean/templates/userinfobox.html

```diff
@@ -17,9 +17,7 @@
 		{/if}
 		{if $rights.info.edit_addr || $rights.info.edit_addr_ack || $rights.info.edit_contact || $rights.info.edit_contact_ack}
 			{if $rights.info.edit_addr || $rights.info.edit_addr_ack || $rights.info.edit_contact || $rights.info.edit_contact_ack}
-				<button id="updatedatabtn" type="button" class="btn btn-primary btn-sm" data-toggle="modal" data-target="#updateuserinfobox" accesskey="A">
-					{trans("Actualize")}
-				</button>
+				{button id="updatedatabtn" class="btn btn-primary btn-sm" data_toggle="modal" data_target="#updateuserinfobox" accesskey="A" icon="reload" label="Actualize"}
 			{/if}
```

Comment:
To może jak juz tak zmieniamy to od razu `Actualize` => `Update`? Dziwnie brzmi `Actualize` po angielsku ;-)

Reply:
https://github.com/lmsgit/lms/pull/1997/files#diff-7e728c696cc0c7aae57428f4e05444f9560e168609443c50ad10ad3ccf664636